vertical-pod-autoscaler/pkg/admission-controller/main.go

```diff
@@ -58,6 +56,9 @@ var (
 
 	port            = flag.Int("port", 8000, "The port to listen on.")
 	address         = flag.String("address", ":8944", "The address to expose Prometheus metrics.")
+	kubeconfig      = flag.String("kubeconfig", "", "Paths to a kubeconfig. Only required if out-of-cluster.")
```

Comment:
s/Paths/path/

Reply:
I went for a capital `Path` for consistency, I guess you were only after the trailing `s` anyway.

Reply:
Hm, I still see the s at the end?

Reply:
Here and in recommender/main.go